### PR TITLE
Implementando lógica y diseño para la creación, edición y visualización de actividades y Agregando name a las rutas

### PR DESCRIPTION
**Pull request by MarioRam0s**
## Description
* Implementando lógica y diseño para la creación, edición y visualización de actividades
* Agregando name a las rutas

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Refactoring (no functional changes, no api changes)
- [ ] This change requires a documentation update
- [ ] Other (please describe): 

## How Has This Been Tested?

llll
lllll
llllll

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in areas hard to understand.
- [ ] I have made the corresponding changes to the documentation.
- [x] My changes do not generate new warnings.
- [x] I have added tests that prove my solution is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] All dependent changes have been merged and published in subsequent modules.
- [x] I have reviewed my code and corrected any spelling mistakes.

### CODE REVIEW

**Comment by SamBarFu** [comment]
Las capturas son de una tablet, cuando sea la orientación de la app sea portrait mejor emular en dispositivos móviles y subir capturas con ese dispositivo

File: lib/presentation/pages/activities/new_activity.page.dart [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+            visible: true,
+            child: ButtonTextIcon(
+              title: S.current.Guardar,
+              icon: const Icon(Icons.save),
+              buttonColor: $colorSuccess,
+              onClic: () {
+                toastAlert(
+                  context: context,
+                  //TODO: Aplicar internacionalizacion luego
+                  title: '¡Guardado con exito!',
```
Comment (SamBarFu) [inline lib/presentation/pages/activities/new_activity.page.dart:38]: Corregir ortografía

File: lib/presentation/pages/activities/new_activity.page.dart [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+              onClic: () {
+                modalDialogConfirmation(
+                  context: context,
+                  titleButtonConfirm: S.current.Si_salir,
+                  question: RichText(
+                    textAlign: TextAlign.center,
+                    text: const TextSpan(
+                      //TODO: Crear variable de internacionalizacion,
+                      text:
+                          'Esta seguro de salir de la creacion de la actividad',
```
Comment (SamBarFu) [inline lib/presentation/pages/activities/new_activity.page.dart:62]: Esto es una pregunta, agregar signos de exclamación y corregir ortografía

**Review by SamBarFu — CHANGES_REQUESTED** [review]

File: lib/presentation/pages/activities/new_activity.page.dart (hunk 1 above)
Comment (LordMario) [reply lib/presentation/pages/activities/new_activity.page.dart]: Listo

File: lib/presentation/pages/activities/new_activity.page.dart (hunk 2 above)
Comment (LordMario) [reply lib/presentation/pages/activities/new_activity.page.dart]: Listo

**Review by SamBarFu — APPROVED** [review]

**Pull request merged**